spec/support/fixtures/integrations/discord/interactions/author.rb

```diff
@@ -0,0 +1,38 @@
+require 'faker'
+require 'json'
+
+module Integrations
+  module Discord
+    module Interactions
+      class Author
+        def initialize
+          @avatar = nil
+          @avatar_decoration_data = nil
+          @clan = nil
+          @discriminator = '0'
+          @global_name = Faker::Internet.username
+          @id = Faker::Number.number(digits: 18).to_s
+          @public_flags = Faker::Number.between(from: 0, to: 10)
+          @username = Faker::Internet.username
+        end
```

Comment:
I think we can use FactoryBot for this. FactoryBot can be used to define factories for all kinds of classes, it doesn't need to be rails models. You can see how we implemented it on structuraid

I think you just need to be explicit with the class like this https://github.com/bro-garden/structuraid-core/blob/2739bea42e8960c5ba74a4578d267edf4cd993ad/spec/factories/elements/footing.rb#L4

And you need to add these lines in the factory definition https://github.com/bro-garden/structuraid-core/blob/2739bea42e8960c5ba74a4578d267edf4cd993ad/spec/factories/elements/footing.rb#L28-L29

Reply:
Uff this is cool! But I was trying to create a fixture of the request params, not for a class, but I really thing we don't need it; please take a look!